js/statistics_grid.js

```diff
@@ -0,0 +1,21 @@
+(function ($, Drupal) {
+
+  'use strict';
+
+  Drupal.behaviors.hideParagraphs = {
```

Comment:
Maybe rename this to describe what it's doing. It's not hiding paragraphs but more about making sure there are at least 2 statistic grid blocks.